app/views/govuk_publishing_components/components/docs/accordion.yml

```diff
@@ -274,6 +274,32 @@ examples:
             text: "How people read"
           content:
             html: "<p class='govuk-body'>This is the content for How people read.</p>"
+  with_id_on_headings:
+    description: |
+      Some apps require custom ids per accordion section heading for linking between those specific sections across multiple pages. An example of this is on manuals pages where multiple manuals will each include large sets of accordions and will reference between specific sections for ease of access to that content.
+
+      This attribute is paired with functionality to automatically open an accordion section on page load where the URL hash is the same as the id of that accordion. [Live example](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/creating-and-updating-pages#associations)
+
+      Unlike with the accordion-wide custom id attribute, this id isn't stored in `localStorage` so does not need to be unique across your domain, but should still be unique to the page.
+    data:
+      items:
+        - heading:
+            text: "Writing well for the web"
```

Comment:
For reference I don't believe you need to put quotes around simple strings like this in yml.

Reply:
If you're ok with it I'll leave this how it is for now as this seems to be the pattern for the rest of the doc, however I'll keep this in mind in future. I can issue it if you want?

Reply:
Yes, that would be great, thanks.